src/app/app.component.ts

```diff
@@ -14,6 +14,8 @@ import { Notifications } from '../providers/notifications';
   templateUrl: './app.html'
 })
 export class StockpileApp {
+  // Need to ViewChild Nav instead of using NavController because this is
```

Comment:
Oh yeah that one is a little odd because it is meant to describe the class variable.